pkg/cloud/aws/services/ec2/vpc_test.go

```diff
@@ -113,7 +113,7 @@ func TestReconcileVPC(t *testing.T) {
 					}, nil)
 
 				m.ModifyVpcAttribute(gomock.AssignableToTypeOf(&ec2.ModifyVpcAttributeInput{})).Return(&ec2.ModifyVpcAttributeOutput{}, nil)
```

Comment:
You should be able to use `.Times(2)` to avoid repetition

Reply:
ah, ok